strongbox-web-core/src/test/java/org/carlspring/strongbox/rest/SearchControllerTest.java

```diff
@@ -34,19 +34,19 @@
     private static final String REPOSITORY_RELEASES = "sc-releases-search";
```

Comment:
@sbespalov : Would you mind restoring the original indentation of the class? It's a bit hard to follow the changes. Thanks! :)